packages/compose/README.md

```diff
@@ -188,7 +188,7 @@ Returns true if the viewport matches the given query, or false otherwise.
 _Usage_
 
 ```js
-useViewportMatch( 'huge', '<' );
+useViewportMatch( 'huge', <' );
```

Comment:
Accidental edit?

Reply:
This typo was in `master` until #20911, so this is probably a rebase mistake.